Description: Define PATH_MAX and MAXPATHLEN for GNU/Hurd compatibility
Author: Anthony Fok <foka@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/977954
Forwarded: no
Last-Update: 2021-01-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/inc/basics.h
+++ b/inc/basics.h
@@ -132,5 +132,13 @@
 			last = newitem;		       \
 		    }
 
+#ifdef __GNU__
+#  ifndef PATH_MAX
+#    define PATH_MAX 4096
+#  endif
+#  ifndef MAXPATHLEN
+#    define MAXPATHLEN 4096
+#  endif
+#endif
 
 #endif /* FONTFORGE_BASICS_H */
